selenium/scripts/check_endpoints.pl

```diff
@@ -0,0 +1,106 @@
+#!/usr/local/bin/perl
+
+=head1 LICENSE
+
+Copyright [2009-2018] EMBL-European Bioinformatics Institute

```

Comment:
This should specify joint Sanger/EMBL-EBI copyright for 1999-2015 and EMBL-EBI one for 2018-2018, shouldn't it?

Reply:
No, my understanding is that EBI-only copyright is correct.

Reply:
But I'm not sure what dates should be used - let me check

Reply:
Hmph. Had a look at _Ensembl Coding Guidelines_ on Confluence, they only mentioned the joint copyright. On the other hand, the _Ensembl Software Licence_ mentions the combined one but without having bumped the year from 2017 to 2018.

I give up.

Reply:
Aha. I changed it to match the existing copyright in the REST repo (i.e. with sanger attribution) for consistency after discussing with @andrewyatz.